docs/dev/triage.md

```diff
@@ -4,357 +4,227 @@ Original RFC: [bit.do/cirq-triage](bit.do/cirq-triage)
+* `help wanted` - If a project lead wants help on a certain task or a high priority item needs to be done but there is no one to assign it yet, we should put the `help wanted` label on it.  
 
-**good for learning** (level/advanced in the areas needed and complexity/low) - the issue is relatively small, self contained, but requires digging into some areas and develop a solid understanding. Should be a bit harder than "good first issues". 
+### Implementation and design
 
-**good part time project -** (level/advanced and complexity/medium)**  **the issue might take up a couple of months, needs a design and multiple conversations, can require digging deep into a couple of papers. It is still self-contained, doesn't have too much dependencies on the rest of Cirq. 
+After an issue arrives to `triage/accepted` there can be two avenues: it is ready to be implemented (most of the cases) or it needs design work upfront. 
 
-**help wanted **- If a project lead wants help on a certain task or a high priority item needs to be done but there is no one to assign it yet, we should put the **help wanted** label on it.  
+When an issue is ready to be implemented, no extra label is required to signal the readiness, that is the default. 
 
-### Implementation and design
-
-After an issue arrives to **triage/accepted**** **there can be two avenues: it is ready to be implemented directly (most of the cases) or it needs design work upfront. The former has no extra label signalling the readiness, that is the default. However when there is a need for design, we add the label **needs design****. **The design could be as lightweight as a discussion in the issue itself or a full fledged RFC proposal which should be clear from the comments. 
+However, when there is a need for design, we add the label `needs agreed design`. The design could be as lightweight as a discussion in the issue itself or a full fledged [RFC proposal](rfc_process.md) which should be clear from the comments. 
 
 ### Assigning work
 
 Assignment should be a function of 
 
-* **priority** (critical issues shouldn't depend on part time work)
-
-* **complexity** (highly complex, large pieces are not feasible/rewarding part time necessarily)
-
-* **skills** - if someone does not have the skills for a given issue, they will have to factor in the learning that's required to do it
-
-* **willingness**- contributors should volunteer to take issues or maintainers should take them actively 
+* **priority** - critical issues shouldn't depend on part time work.
+* **complexity** - highly complex, large pieces are not feasible/rewarding part time necessarily.
+* **skills** - if someone does not have the skills for a given issue, they will have to factor in the learning that's required to do it.
+* **willingness** - contributors should volunteer to take issues or maintainers should take them actively.
```

Comment:
Recommend moving this to the top - if someone doesn't want to take an issue, we can't/won't force them to take it.